doc/foresight.rst

```diff
@@ -163,109 +190,152 @@ technologies.
-   step are added if they are still in operation (i.e., if they fulfil planning
-   horizon <= commissioned year + lifetime). In addition, the network comprises
-   additional generator, storage, and link capacities with
-   p_nom_extendable=True. The non-solved network is saved in
-   ``results/run_name/networks``.
+**Example 1: Per-period caps with fraction values**
 
-Steps 2 and 3 are solved recursively for all the planning_horizons included in
-``config/config.yaml``.
+.. code:: yaml
 
-Rule overview
---------------
+  co2_budget:
+    values: fraction
+    upper:
+      2030: 0.450  # 45% of 1990 emissions
+      2040: 0.100
+      2050: 0.000
+
+**Example 2: Scalar cap (absolute values)**
+
+.. code:: yaml
+
+  co2_budget:
+    values: absolute
+    upper: 2.04  # Gt CO2/year, same semantics across foresight modes (see above)
+
+**Example 3: Selective year constraints**
 
-- rule add_existing baseyear
+.. code:: yaml
+
+  planning_horizons: [2030, 2040, 2050]
+
+  co2_budget:
+    values: absolute
+    upper:
+      2030: 2.0
+      2050: 0.0
+      # 2040 not specified -> no upper constraint for 2040
+    lower:
+      2030: 0.5
+
+**Emissions scope:**
+
+The ``emissions_scope`` parameter determines which greenhouse gas(es) are accounted for
+when calculating the 1990 baseline for ``values: fraction`` and applying the corresponding
+budget constraints. This parameter corresponds to the ``Pollutant_name`` field in the EEA
+UNFCCC emissions database.
+
+Available options:
+
+- ``All greenhouse gases - (CO2 equivalent)`` - All greenhouse gases in CO₂-equivalent,
+  including CO₂, CH₄, N₂O, and fluorinated gases (HFCs, PFCs, SF₆, NF₃) (**default**)
+- ``CO2`` - Carbon dioxide only
+- ``CH4`` - Methane emissions only
+- ``N2O`` - Nitrous oxide emissions only
+
+The choice of emissions scope affects:
+
+1. **Baseline calculations**: Historical emissions used to calculate budget fractions
+   (e.g., "55% reduction from 1990" requires knowing 1990 emissions)
+2. **Constraint application**: The optimization constraints in the network model apply
+   to the selected emissions scope
 
-  The rule add_existing_baseyear loads the network in
-  ``resources/run_name/networks`` and performs the following operations:
+.. note::
+   The default uses all greenhouse gases in CO₂-equivalent to align with EU climate policy
+   targets (e.g., Fit for 55: 55% GHG reduction by 2030, climate neutrality by 2050). This
+   ensures budget calculations use the same baseline as official climate targets. While the
+   energy system model primarily tracks and optimizes CO₂ emissions from energy use, using
+   all GHGs for baseline calculations provides proper context for the model's emission
+   constraints relative to economy-wide climate goals. Non-energy GHG emissions (e.g., from
+   agriculture, industrial processes) are typically handled as exogenous assumptions or
+   boundary conditions.
```

Comment:
Double-check. AFAIK, PyPSA-Eur only covers CO2 emissions, not CO2 equivalents...